db/migrate/20231024194120_add_index_on_route_param_for_admin_action_logs.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddIndexOnRouteParamForAdminActionLogs < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :admin_action_logs, [:route_param], algorithm: :concurrently
+  end
+end
```

Comment:
I think this should be a post deployment migration. Building the index might take a while but the application will probably work fine without it.